packages/manager/apps/web/client/app/hosting/multisite/git-association/hosting-multisite-git-association.controller.js

```diff
@@ -25,27 +27,40 @@ export default class HostingMultisiteGitAssociationController {
       webhookUrl: '',
     };
     this.coreURLBuilder = coreURLBuilder;
+    this.$translate = $translate;
   }
 
   $onInit() {
-    this.errorMessage = !this.sshKey;
     this.ongoingTasksHref = this.coreURLBuilder.buildURL(
       'web',
       `#/hosting/${this.serviceName}/task`,
     );
+    this.model.webhookUrl = this.webHookUrl;
```

Comment:
Clearer if the initialization is made at only one location (you can remove it from the constructor)
```suggestion
    this.model = {
     webhookUrl: this.webHookUrl,
     repositoryUrl: null,
     repositoryUrl: null,
     }
```